docs/reporting.md

```diff
@@ -187,4 +187,73 @@ There are basically two things to keep in mind when upgrading to `SoberSwag::Rep
    Instead, view management is now *explicit*.
    This is because it was too tempting to pass data to serialize in the options key, which is against the point of the serializers.
 
-
+## API Overview
+
+This section presents an overview of the available reporting outputs and inputs.
+
+### `SoberSwag::Reporting::Output`
+
+This module contains reporting *outputs*.
+These act as type-checked serializers.
+
+#### Primitive Types
+
+The following "primitive types" are available:
+
+- `SoberSwag::Reporting::Output.bool`, which returns a `SoberSwag::Reporting::Output::Bool`.
+  This type is for serializing boolean values, IE, `true` or `false`.
+  It will serialize the boolean directly to the JSON.
+- `SoberSwag::Reporting::Output.null`, which returns a `SoberSwag::Reporting::Output::Null`.
+  This type serializes out `null` in JSON.
+  This can only serialize the ruby value `nil`.
+- `SoberSwag::Reporting::Output.number`, returns a `SoberSwag::Reporting::Output::Number`.
+  This type serializes out numbers in JSON.
+  It can serialize out most ruby numeric types.
```

Comment:
Should this mention how to handle floats? (I think you're supposed to use to_f, but maybe not?

Reply:
It actually will serialize floats properly directly.